__tests__/app.test.js

```diff
@@ -5,7 +5,11 @@ const data = require('../db/data/test-data')
 const db = require('../db/connection')
 const endpoints = require('../endpoints.json')
 
-beforeEach(() => seed(data));
+beforeEach(() => {
+    return db.query('DELETE FROM comments;')
+        .then(() => db.query('DELETE FROM articles;'))
+        .then(() => seed(data))
+})
```

Comment:
Not sure what these db.queries are doing, don't worry about deleting anything every time you run the seed the tables are reset anyway. Get rid of these and just have:

beforeEach(() => seed(data))